lib/led/digits.js

```diff
@@ -0,0 +1,13 @@
+var LedControl = require("./ledcontrol"),
+  _ = require("lodash");
+
+// stub implementation; extract functionality from ledcontrol.js
+var Digits = function(opts) {
+  opts.isMatrix = false;
+  return new LedControl(opts);
+};
+_.extend(Digits, LedControl, {
```

Comment:
Out of curiosity then, when is LoDash "allowed"?  What's its purpose?  If we just need one or two functions, can we cherry-pick?